compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIELowerWorkgroupCount.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#include "iree-amd-aie/Transforms/PassDetail.h"
+#include "iree-amd-aie/Transforms/Passes.h"
+#include "iree/compiler/Dialect/Flow/IR/FlowOps.h"
+#include "iree/compiler/Dialect/HAL/IR/HALOps.h"
+#include "mlir/IR/PatternMatch.h"
+#include "mlir/Pass/Pass.h"
+#include "mlir/Support/LogicalResult.h"
+
+namespace mlir::iree_compiler::AMDAIE {
+
```

Comment:
1. Better to add a `DEBUG_TYPE` ? Although it's needed when guarding a debug statement with `LLVM_DEBUG`, it helps know the actual pass name that can be invoked via command-line.

2. Also, would it be favourable to add a lit test for this now that a separate pass is created ? (We anyway don't have a separate pass testing the cleanup pass so perhaps we can skip this - just adding a query)

Reply:
I think I already added  a lit test?

Reply:
Ouch. My bad! Just checked again. Sounds good.